content/html/concepts/semantic-html/terms/ruby-text/ruby-text.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: '<rt>'
```

Comment:
Looks like this path is incorrect. The folder and filename should use the tag, "rt". So, it should be:
.../terms/rt/rt.md